corehq/apps/data_cleaning/models.py

```diff
@@ -103,6 +103,30 @@ def percent_complete(self):
             return None
         return round(self.result['percent'])
 
+    @property
```

Comment:
Suggestion: I think it's easy to confuse `has_any_filtering` and `has_filters`, or `reset_filtering` and `reset_filters`. Users are always going to reset both pinned and regular filters together, right? What would you think of inlining the helper functions, so that there's just one filter check function and one filter reset function, both of which handle both types of filters?

Reply:
i was imagining a prompt (later) that lets users choose which filters they want to reset when they click on the X in the toolbar, especially inspired from Daniel's questions later. this can be addressed later after the feature is released.